intranethome/views.py

```diff
@@ -1,239 +1,246 @@
-from argparse import REMAINDER
-from ast import Name, alias
+import json
+import re
+from functools import cmp_to_key
 from base.wagtail_hooks import (
     get_required_groups,
     has_permission,
     redirect_users_without_permissions,
 )
-from multiprocessing import process
-from django.http import HttpResponse
 from django.shortcuts import render
-from functools import cmp_to_key
 from library_website.settings import MAIL_ALIASES_PATH
-import json
-import re
+from site_settings.models import ContactInfo
 from wagtail.models import Site
 
+parse_error_message = {
+    "error": {
+        "link_url": ContactInfo.objects.first().report_a_problem,
+    }
+}
```

Comment:
This might be wrong, since I see there's also a problem with the migrations that's related. I guess we'll see after we sort that out if this still fails in CI.

Reply:
UPDATE: The problem is actually the whole `ContactInfo` bit. This is also what's blowing up Vagrant builds. It never exists on a new site so it blows things up.